#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

gen = ParameterGenerator ()

gen.add("delay_time", double_t, 0, "Delay time of input point cloud", 1.0, 0.0, 100.0)

exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "DelayPointCloud"))
